duckduckgo/Android
# Disconnect List Blocking and Tracker Network Summary

Asana Issue URL: 
Disconnect List Blocking: https://app.asana.com/0/414235014887631/471718733961916
Tracker Networks Summary: https://app.asana.com/0/414235014887631/364367284740782

## Description
• Downloads the disconnect list from the server
• Adds disconnect list tracker blocking
• Adds a skeleton privacy grade page
• Displays tracker network count (currently static, live updates will be added in future tasks)

NOTE: Data sync service and storage and live are covered by separate tasks

## Steps to Test this PR:
Test One:
1. Navigate to duckduckgo.com
1. Open the Privacy Dashboard
1. Once loading has finished, observe that the major tracker number is 0

Test two:
1. Navigate to evanscycles.com
1. Open the Privacy Dashboard
1. Once loading has finished, observe that the major tracker number is 6
